api/apps/api/src/modules/clone/import/application/import-piece-failed.event.ts

```diff
@@ -0,0 +1,12 @@
+import { ComponentId, ResourceId, ResourceKind } from '@marxan/cloning/domain';
+import { IEvent } from '@nestjs/cqrs';
+import { ImportId } from '../domain';
+
+export class ImportPieceFailed implements IEvent {
```

Comment:
I think we should only need `importId`and `componentId` as we can query the rest of the data